jenkins-jobs/docker/debezium-testing-system/downstream/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM testsuite-base:latest
+
+COPY apicurio-registry-install-examples.zip /testsuite/apicurio.zip
+COPY amq-streams-install-examples.zip /testsuite/strimzi.zip
```

Comment:
I would recommend to use multi-staged docker build instead of having multiple dockerfiles. https://docs.docker.com/develop/develop-images/multistage-build/